language-server/modules/langserver-core/src/test/resources/codeaction/change-var-type/source/changeVarType_in_select_clause.bal

```diff
@@ -0,0 +1,17 @@
+type Person record {|
+    string firstName;
+    string lastName;
+    int age;
+|};
+function testFunction() {
+    Person p1 = {firstName: "Alex", lastName: "George", age: 33};
+    Person p2 = {firstName: "John", lastName: "David", age: 35};
+    Person p3 = {firstName: "Max", lastName: "Gomez", age: 33};
+    Person[] personList = [p1, p2, p3];
```

Comment:
You can use same test case with two query expression to avoid duplication of data

Reply:
Used the same test case